scripts/lychee.js

```diff
@@ -141,6 +142,7 @@ lychee.init = function() {
 			lychee.image_overlay_default= (data.config.image_overlay && data.config.image_overlay === '1')			|| false;
 			lychee.image_overlay		= lychee.image_overlay_default;
 			lychee.image_overlay_type	= data.config.image_overlay_type;
```

Comment:
you need to do something similar as in line 142:
```js
lychee.image_overlay_type	= (!data.config.image_overlay_type) ? 'exif' : data.config.image_overlay_type;
```

Also, this also need to be done around line 175 (one if you are logged in, one if you are not). :)